src/main/kotlin/com/exactpro/th2/common/schema/message/configuration/MessageRouterConfiguration.kt

```diff
@@ -92,7 +92,8 @@ enum class FieldFilterOperation {
     NOT_EQUAL,
     EMPTY,
     NOT_EMPTY,
-    WILDCARD
+    WILDCARD,
+    NOT_WILDCARD,
```

Comment:
This may not be enough. I see that there is a [JSON schema](https://github.com/th2-net/th2-infra/blob/master/chart/crds/th2-box-crd.yaml#L406) in the infra repo which describes the operation field as an enum.
We should check if this schema is actually applied and modify it as well if required

Reply:
Thank you for remind, as I remember from the last discussion we going to provide ability for customizing CRD for users.
I will add change into th2-infra repository too